mmv1/products/vertexai/FeatureOnlineStore.yaml

```diff
@@ -0,0 +1,225 @@
+    name: region
+    description: The region of the dataset. eg us-central1
+    url_param_only: true
+    immutable: true
+    default_from_api: true
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    description:
+      The name of the FeatureOnlinestore. This value may be up to 60 characters, and
+      valid characters are [a-z0-9_]. The first character cannot be a number.
+    immutable: true
+    url_param_only: true
+    pattern: projects/{{project}}/locations/{{region}}/featureOnlineStores/{{name}}
+    custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
+  - !ruby/object:Api::Type::String
+    name: 'etag'
+    description: Used to perform consistent read-modify-write updates.
+    output: true
+    ignore_read: true
+  - !ruby/object:Api::Type::String
+    name: 'createTime'
+    output: true
+    description: |
+      The timestamp of when the featurestore was created in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+  - !ruby/object:Api::Type::String
+    name: 'updateTime'
+    output: true
+    description: |
+      The timestamp of when the featurestore was last updated in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: 'labels'
+    description: |
+      A set of key/value label pairs to assign to this Featurestore.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'dedicatedServingEndpoint'
+    description: |
+       The dedicated serving endpoint for this FeatureOnlineStore, which is different from common vertex service endpoint. Only need to set when you choose Optimized storage type or enable EmbeddingManagement. Will use public endpoint by default.
+    min_version: beta
+    default_from_api: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'publicEndpointDomainName'
+        description: |
+          populated with the domain name to use for this FeatureOnlineStore
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'serviceAttachment'
+        description: |
+          name of the service attachment resource. Populated if private service connect is enabled and after FeatureViewSync is created.
+        output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'privateServiceConnectConfig'
+        description: |
+          Private service connect config.
+        default_from_api: true
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'enablePrivateServiceConnect'
+            required: true
+            default_from_api: true
+            description: |
+              If set to true, customers will use private service connection to send request. Otherwise, the connection will set to public endpoint.
+          - !ruby/object:Api::Type::Array
+            name: 'projectAllowlist'
+            default_from_api: true
```

Comment:
Removed the field here as well. I thought the default here would be an empty array from the API if not provided because the field is not required but I do now think after running some API calls that there is no API response for that field.